src/components/NavBar /ShoppingCartItem.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import SectionSeperator from "./SectionSeperator";
+
+const ShoppingCartItem = ({ planName, num1, num2 }) => {
```

Comment:
Be a bit more descriptive with naming the props. num1 and num2 could be improved.